users/models.py

```diff
@@ -23,37 +23,19 @@ class Meeting(TimeStampModel):
     class Meta: 
         db_table = 'meetings'
 
-class DrinkingMethod(models.Model): 
-    name = models.CharField(max_length=10)
-
-    class Meta: 
-        db_table = 'drinking_methods'
-        
 class UserDrinkingMethod(models.Model): 
     drinking_method = models.ForeignKey('DrinkingMethod', on_delete=models.CASCADE)
     user            = models.ForeignKey('User', on_delete=models.CASCADE)
 
     class Meta: 
         db_table = 'user_drinking_methods'
 
-class AlcoholLevel(models.Model): 
+class DrinkingMethod(models.Model): 
     name = models.CharField(max_length=10)
+    user = models.ManyToManyField('User', through=UserDrinkingMethod)
```

Comment:
ManyToManyField를 사용하실 때는 column을 복수로 만들어 주셔야 합니다.
그래야 아래와 같이 가독성 있는 코드를 작성할 수 있습니다.
```python
class Product(models.Model) :
    tags = models.ManyToManyField('Tag', through = 'ProductTag')

product = Product.objects.get()
..

tags = product.tags.all()
```
```suggestion
    users = models.ManyToManyField('User', through=UserDrinkingMethod)
```

Reply:
상세한 설명 감사합니다~! 수정완료하였습니다!